bin/recovery.c

```diff
@@ -47,7 +47,14 @@ extern char  *DataDir;
 /* Entry point for the recovery. */
 extern int	LoaderRecoveryMain(void);
 
-static void GetSegmentPath(char path[MAXPGPATH], RelFileNode rnode, int segno);
+
+static void GetSegmentPath(char path[MAXPGPATH], 
+#if PG_VERSION_NUM >= 160000
+												RelFileLocator relNumber, 
```

Comment:
Since relNumber is a member of RelFileLocator, is it better s/relNumber/rlocator/g? Be careful that relNumber exists in other codes.

Reply:
I changed relNumber to rLocator